ONSdigital/dp-search-query
# Replace logging library with log.go

### What

Replace logging library with log.go

### How to review

* Check there are no usages of go-ns/log
* Check logging is appropriate
* Check logging levels

### Who can review

Anyone but me
